Sources/NIO/RecvByteBufferAllocator.swift

```diff
@@ -89,78 +66,50 @@ public struct AdaptiveRecvByteBufferAllocator: RecvByteBufferAllocator {
         precondition(initial > minimum, "initial: \(initial)")
         precondition(maximum > initial, "maximum: \(maximum)")
```

Comment:
Why can't `initial` and `maximum` be equal to `initial`?

Reply:
No particular reason, but I wasn't planning to lift that constraint today.